docs/_data/docs.yml

```diff
@@ -6,6 +6,7 @@
   - usage
   - structure
   - configuration
+  - windows
```

Comment:
We could keep the `Jekyll on Windows` title and put it under the installation entry